### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by Jjkrules**
created a screen 0, 4, and 5 for the AI Single Player portion of the game still need to add the AI moves and board to screen 5 for now it prints that it accepts user input from screen 4.

### CODE REVIEW

**Review by sabioe — APPROVED** [review]
looks nice

**Comment by chris-douglas13** [comment]
conflict fixed waiting for approval

**Pull request closed without merging**
